docs/design/analytics.md

```diff
@@ -25,94 +25,86 @@ One dimension per flag, and although technically there can be an overlap between
 simplicity it should remain unique across all CLI commands. The dimension is the value of the
 `x-user-analytics` field in the `schema.json` files.
 
-To create a new dimension (tracking a new flag):
-
-1. Create the dimension on analytics.google.com first. Dimensions are not tracked if they aren't
-   defined on GA.
+### Adding dimension or metic.
+1. Create the dimension or metric in (https://analytics.google.com/)[Google Analytics] first. These are not tracked if they aren't
+   defined in Google Analytics.
 1. Use the ID of the dimension as the `x-user-analytics` value in the `schema.json` file.
-1. Add a new row to the table below in the same PR as the one adding the dimension to the code.
-1. New dimension PRs need to be approved by the tooling and DevRel leads.
-   **This is not negotiable.**
+1. New dimension and metrics PRs need to be approved by the tooling lead and require a new (http://go/launch)[Launch].
+
+### Deleting a dimension or metic.
+1. Archive the dimension and metric in (https://analytics.google.com/)[Google Analytics].
+
 
 **DO NOT ADD `x-user-analytics` FOR VALUES THAT ARE USER IDENTIFIABLE (PII), FOR EXAMPLE A
 PROJECT NAME TO BUILD OR A MODULE NAME.**
 
-Note: There's a limit of 20 custom dimensions.
+### Limits
+| Item                           	| Standard property limits 	|
+|--------------------------------	|--------------------------	|
+| Event-scoped custom dimensions 	| 50                       	|
+| User-scoped custom dimensions  	| 25                       	|
+| All custom metrics             	| 50                       	|
 
-### List Of All Dimensions
+### List of User Custom Dimensions
```

Comment:
@mgechev since @dgp1130 is OOO can you please take a look the new list of data collected?